java/lang/Float/Float.hpp

```diff
@@ -176,33 +177,12 @@ namespace Java {
 			static constexpr int BYTES = SIZE / 8;
 
 		public:
-			/**
-       * Float initialization
-       *
-       * @param original
-       */
 			Float();
-
-			/**
-       * Float initialization
-       *
-       * @param original
-       */
 			Float(float original);

```

Comment:
Please add comment

Reply:
Thanks, I will dicuss with team about this.

Reply:
Agreed. Fixed.